### PR TITLE
Added variable date range to dashboard.

### PR DESCRIPTION
**Pull request by marcoau**

### CODE REVIEW

File: app/modules/dashboard/dashboardGraphics.js [hunk 1]
```diff
@@ -10,17 +10,18 @@ angular.module('dashboardGraphics', [])
         var users = data[0];
         var scores = data[1];
         var averages = data[2];
+        console.log(averages);
```
Comment (vhalbwachs) [inline app/modules/dashboard/dashboardGraphics.js]: Kenny Loggins

File: app/views/partials/home.html [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+
+    <h1>Welcome To my2cents</h1>
+  </div>
 </div>
 
 <div class="container">
   <div class="row marketing">
   <h4>Metrics</h4>
-  <p>We provide <strong>Metrics</strong></p>
+  <!-- <p>We provide <strong>Metrics</strong></p> -->
```
Comment (vhalbwachs) [inline app/views/partials/home.html:13]: We can just remove this, no need to comment out

File: app/modules/dashboard/dashboardGraphics.js [hunk 3]
```diff
@@ -362,10 +324,13 @@ angular.module('dashboardGraphics', [])
                 var diff = dx + dy;
 
                 //redness and blueness of dot
-                var redness = Math.max(0, Math.min(255, (100-diff) * (255/200)));
-                var blueness = Math.max(0, Math.min(255, (diff+100) * (255/200)));
+                // var redness = Math.max(0, Math.min(255, (100-diff) * (255/200)));
+                // var blueness = Math.max(0, Math.min(255, (diff+100) * (255/200)));
```
Comment (vhalbwachs) [inline app/modules/dashboard/dashboardGraphics.js]: Remove commented out code

File: app/modules/dashboard/dashboardGraphics.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 angular.module('dashboardGraphics', [])
 
-  .factory('mainChart', ['$window', 'fisheyeChart', 'SmoothAverages', 'timeFormat', function($window, fisheyeChart, SmoothAverages, timeFormat){
+  .factory('mainChart', ['$window', 'fisheyeChart', 'Datestamp', 'Scorestamp', 'SmoothAverages', 'PathMethods' ,'TimeFormat', function($window, fisheyeChart, Datestamp, Scorestamp, SmoothAverages, PathMethods, TimeFormat){
```
Comment (omkarv) [inline app/modules/dashboard/dashboardGraphics.js]: Rewrite to make it cover two lines

**Pull request merged**